pypeit/spectrographs/p200_dbsp.py

```diff
@@ -88,6 +88,26 @@ def compound_meta(self, headarr: List[fits.Header], meta_key: str):
         else:
             return None
 
+    def config_independent_frames(self):
+        """
+        Define frame types that are independent of the fully defined
+        instrument configuration.
+
+        This method returns a dictionary where the keys of the dictionary are
+        the list of configuration-independent frame types. The value of each
+        dictionary element can be set to one or more metadata keys that can
+        be used to assign each frame type to a given configuration group. See
+        :func:`~pypeit.metadata.PypeItMetaData.set_configurations` and how it
+        interprets the dictionary values, which can be None.
+
+        Returns:
+            :obj:`dict`: Dictionary where the keys are the frame types that
+            are configuration-independent and the values are the metadata
+            keywords that can be used to assign the frames to a configuration
+            group.
+        """
+        return {'bias': ['binning', 'dichroic'], 'dark': ['binning', 'dichroic']}
```

Comment:
Why does the dichroic matter, or this just a useful proxy for something?

Reply:
I just noticed that this was in the `configuration keys`, and given that this is a double arm spectrograph, I wonder what is determining the red vs blue camera, in this case 🤷‍♂️ I've removed the dichroic, in any case...